src/components/modals/shared/AmountInput.js

```diff
@@ -24,11 +25,12 @@ export default class AmountInput extends Component {
   }
 
   setAmount = event => {
+    // TODO: use freeAll if user inputs the max amount?
```

Comment:
@jparklev do we want to call free all when the user manually enters the max amount?

Reply:
good question. My feeling is that we should. What do you think?

Reply:
Seems like good practice, and I can't think of any particular reason not to do so.